ref/gl/gl_local.h

```diff
@@ -388,7 +388,7 @@ void R_TextureReplacementReport( const char *modelname, int gl_texturenum, const
 void CL_RunLightStyles( lightstyle_t *ls );
 void R_PushDlights( void );
 void R_GetLightSpot( vec3_t lightspot );
-void R_MarkLights( dlight_t *light, int bit, mnode_t *node );
+void R_MarkLights( const dlight_t *light, int bit, const mnode_t *node );
```

Comment:
Sorry for that, I had to fix the commit description and force-pushed the changes.

Cherry-pick your latest commit on top of current `master`.